### ISSUE TITLE
Unable to save edits to existing widgets

### ISSUE THREAD

**Issue by tellthemachines** [WordPress/gutenberg#27747]
**Describe the bug**

In the new Widgets screen, updating the content of legacy text or HTML widgets doesn't enable the "Update" button, so changes can't be saved.

**To reproduce**
1. Go to the legacy Widgets screen.
2. Add a text or a custom HTML widget and save changes.
3. Switch to the new screen and try to edit the content of that widget.
4. Observe "Update" button still disabled.

**Expected behavior**

Expected to be able to save any edits in the new screen.

**Editor version (please complete the following information):**
Latest Gutenberg.

**Comment by talldan** [comment]
Some previous work on a similar issue - https://github.com/WordPress/gutenberg/pull/26144

**Comment by Azragh** [comment]
I can't save anything on the widgets screen. No matter what I do, when I hit save it only says: "An error occured. [object object]" .. 

I found out that I can save legacy widgets when I remove "add_theme_support( 'customize-selective-refresh-widgets' )" from functions.php, but adding blocks still doesn't work.

Tested with several themes without other plugins than gutenberg enabled - the only themes that worked so far are the native ones.
